keras_rs/src/utils/tpu_test_utils.py

```diff
@@ -0,0 +1,170 @@
+import contextlib
+import os
+import threading
+from types import ModuleType
+from typing import Any, Callable, ContextManager, Optional, Tuple, Union
+
+import keras
+import tensorflow as tf
+
+jax: Optional[ModuleType] = None
+
+try:
+    import jax
+except ImportError:
+    pass
+
+
+class DummyStrategy:
+    def scope(self) -> ContextManager[None]:
+        return contextlib.nullcontext()
+
+    @property
+    def num_replicas_in_sync(self) -> int:
+        return 1
+
+    def run(self, fn: Callable[..., Any], args: Tuple[Any, ...]) -> Any:
+        return fn(*args)
+
+    def experimental_distribute_dataset(
+        self, dataset: Any, options: Optional[Any] = None
+    ) -> Any:
+        del options
+        return dataset
+
+
+class JaxDummyStrategy(DummyStrategy):
+    @property
+    def num_replicas_in_sync(self) -> Any:
+        if jax is None:
+            return 0
```

Comment:
I would remove this and simply `import jax` here. It should fail if jax is missing.